apps/quickstart-integrations-monday-code/src/services.js

```diff
@@ -0,0 +1,79 @@
+import initMondayClient from "monday-sdk-js";
+import jwt from "jsonwebtoken";
+import { getSecret } from "./helpers.js";
+
+export const authorizeRequest = (req, res, next) => {
```

Comment:
I think we should mention here that this is a middleware.
In general I think this should be in a middleware file